wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/README.md

```diff
@@ -28,25 +29,43 @@ Follow these steps to set up a local playground for the theme:
 
 2.  Build the MU plugins.
 
-    1.  Clone the `WordPress/wporg-mu-plugins` repo right into this directory.
-        It's `.gitignored` so it shouldn't affect anything.
+    1.  The mu-plugins are set up as composer dependencies, so install those:
         ```bash
-        $ git clone https://github.com/WordPress/wporg-mu-plugins.git
+        $ composer install
         ```
-    2.  `cd` into this directory at `./wporg-mu-plugins`
-        (i.e. `wordpress.org/wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/wporg-mu-plugins`).
+    2.  `cd` into the directory at `./mu-plugins/wporg-mu-plugins`
+        (i.e. `wordpress.org/wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/mu-plugins/wporg-mu-plugins/`).
     3.  Install all the required `npm` packages.
         ```bash
         $ npm install
         ```
-    4.  Build the theme assets.
+    4.  Build the plugin assets.
         ```bash
         $ npm run build
         ```
-    5.  `cd` back to the Openverse theme directory at `..`
+    5.  `cd` back to the Openverse theme directory at `../..`
         (i.e. `wordpress.org/wordpress.org/public_html/wp-content/themes/pub/wporg-openverse`).
 
-3.  You can choose to set up a new environment automatically or work in an
+3.  Set up the locale database. The plugin was installed in the previous step, but it pulls from a separate database of locale data.
+
+    1.  Download the SQL file [wporg_locales.sql](https://raw.githubusercontent.com/WordPress/pattern-directory/trunk/.wp-env/data/wporg_locales.sql) to the theme directory.
```

Comment:
```suggestion
    1.  Download the SQL file [wporg_locales.sql](https://raw.githubusercontent.com/WordPress/pattern-directory/trunk/.wp-env/data/wporg_locales.sql) to the theme directory.
    ```bash
    curl -O https://raw.githubusercontent.com/WordPress/pattern-directory/trunk/.wp-env/data/wporg_locales.sql
    ```
```

Could be nice to suggest a download command, unless we think it's too risky to suggest curl-ing random files.

Reply:
Could also just include the file in the repo somewhere. It doesn't change very often, and it probably doesn't need to be up to date for most testing purposes anyway.